app.py

```
@@ -27,24 +28,36 @@ def resource_path(relative_path):
     return os.path.join(base_path, relative_path)
 
 
+# Configure logging
+logging.basicConfig(
+    level=logging.DEBUG,
+    format='%(asctime)s - %(levelname)s - %(message)s',
+    handlers=[
+        logging.FileHandler('rsteye.log'),
+        logging.StreamHandler()
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)